NEWS

```diff
@@ -11,9 +11,98 @@
 ### Features:
 ### Bugfixes:
 
-## 1.17.0-rc1 (April 19)
-TBD
-
+## 1.17.0-rc2 (May 16, 2024)
+### Features:
+#### UCP
+* Improved performance calculation for rendezvous protocol
+* Enabled short protocol for non-host memory types on empty messages
+* Improved performance estimation for empty messages by removing non-relevant overheads
+* Added RMA_ZCOPY_MAX_SEG_SIZE configuration parameter to allow modifying segment size for RMA-ZCOPY protocols
+* Added support for separate intra/inter-node rendezvous thresholds
+* Added support for minimal fragment size in rendezvous protocol
+* Added support for protocol variants
+* Added support for resetting request during send operation
+* Added UCX_PROTO_OVERHEAD configuration variable to allow setting protocol overheads
+* Improved performance for combined Active Message/RMA scenarios by separating them to different lanes
+* Added support for device staging buffers in pipeline protocols
+* Enable ODP on Grace platforms by default 
+#### RDMA CORE (IB, ROCE, etc.)
+* Introduced the UCX_REVERSE_SL environment variable to configure reverse SL for DC transport. By default, it uses UCX_IB_SL.
+* Added support for GID auto-detection in Floating LID based routing
+* Imporved performance of RC transport when hardware tag matching is not available
```

Comment:
Improved
also what is the change mentioned here?

Reply:
https://github.com/openucx/ucx/pull/9644

Reply:
i'd remove that - it is internal refactoring needed for implementing gga support. No perf improvements expected with this patch